src/main/scala/com/danielasfregola/twitter4s/http/oauth/OAuth1Provider.scala

```diff
@@ -67,7 +67,7 @@ private[twitter4s] class OAuth1Provider(consumerToken: ConsumerToken, accessToke
       val method = request.method.name.urlEncoded
       val baseUrl = request.uri.endpoint.urlEncoded
       val oauthParams = oauth2Params.map {
-        case (k, v) =>
+        case (k: String, v: String) =>
           if (k == "oauth_callback") k -> v.urlEncoded
           else k -> v
       }
```

Comment:
Scala 2.13 Maps actually have a new [`updatedWith()`](https://www.scala-lang.org/api/current/scala/collection/immutable/Map.html#updatedWith[V1%3E:V](key:K)(remappingFunction:Option[V]=%3EOption[V1]):CC[K,V1]) method for performing this operation! 
So when support for 2.11 & 2.12 is dropped, this code can become a bit smaller: :sparkles: 

```scala
val oauthParams = oauth2Params.updatedWith("oauth_callback")(_.map(_.urlEncoded))
```

Reply:
Thanks, I'll keep it in mind when 2.11 & 2.12 will no longer be in the picture :)